strr-base-web/app/components/modal/help/RegisterStr.vue

```diff
@@ -4,11 +4,11 @@
       <i18n-t keypath="modal.help.registerStr.content" tag="p" scope="global">
         <template #link>
           <a
-            href="https://www.STRregistry.gov.bc.ca"
+            href="https://www.gov.bc.ca/STRRegistry"
```

Comment:
Not sure if this is pending, but that site doesn't exist.
Could we pass it in as an envvar?